frontend/src/components/errors/NotFoundPage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useTranslation } from 'next-i18next'
+import { useRouter } from 'next/router'
+
+import { Button, Grid, Typography } from '@material-ui/core'
+import { createStyles, Theme, makeStyles } from '@material-ui/core/styles'
+import ArrowBackIcon from '@material-ui/icons/ArrowBack'
+
+import Layout from 'components/layout/Layout'
+import NotFoundIllustration from './assets/NotFoundIllustration'
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+      height: '100vh',
+      margin: 0,
+      backgroundColor: theme.palette.secondary.light,
+    },
+  }),
+)
+
+export default function NotFoundPage() {
+  const { t } = useTranslation()
+  const router = useRouter()
+
+  const classes = useStyles()
+
+  return (
+    <Layout maxWidth={false} disableOffset disableGutters>
+      <Grid container className={classes.root} xs={12}>
+        <Grid container direction="column" alignItems="center" justify="center" spacing={4} xs={12}>
+          <Grid container item xs={8} sm={6} lg={4}>
+            <NotFoundIllustration />
+          </Grid>
+          <Grid item>
+            <Typography variant="body2">{t('errors:404Message')}</Typography>
+          </Grid>
+          <Grid container item alignItems="center" justify="center">
+            <Button
```

Comment:
Maybe we can use `<LinkButton />` from `components/common/LinkButton.tsx` instead of plain button with `onClick` handler as it will render a more standard `<a href />` component

At the moment the back button is not functional until the page is completely rendered.

Reply:
Good idea, first I tried with the `<LinkButtonIcon />` and I saw that it is rendering something that I don't need. :D

But now I realized that `<LinkButton />` will do the job, thanks! :)

Reply:
Awesome! I think we're good to go :tada: 
Let's roll